procrustes/test/test_psdp.py

```diff
@@ -85,4 +98,74 @@ def test_psdp_generic_non_square():
         ]
     )
     assert_almost_equal(s, actual_result, decimal=5)
-    assert_almost_equal(error, 5.674530443833204, decimal=3)
+    assert_almost_equal(error, 32.200295757989856, decimal=3)
+    res = psdp_peng(a=a, b=b)
+    s, error = res["s"], res["error"]
+    actual_result = np.array(
+        [
+            [2.58773004, 1.10512076, -1.07911235],
+            [1.10512076, 1.65881535, 0.14189328],
+            [-1.07911235, 0.14189328, 0.75610083],
+        ]
+    )
+    assert_almost_equal(s, actual_result, decimal=5)
+    assert_almost_equal(error, 32.21671819685297)
+
+
+def test_psdp_non_full_rank():
```

Comment:
The same as above.

Reply:
fixed in [8c7015b](https://github.com/theochem/procrustes/pull/187/commits/8c7015b5413c07718ecf7cd817f58568144fa3ae)